src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,70 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
```

Comment:
Just noticed this. Stack is not a good name. Your class is stack it makes some stack related operations so how this array can be a stack ? It is something else rename it.